docs/rp2/quickref.rst

```diff
@@ -187,6 +187,13 @@ Use the :ref:`machine.ADC <machine.ADC>` class::
     adc = ADC(Pin(26))     # create ADC object on ADC pin
     adc.read_u16()         # read value, 0-65535 across voltage range 0.0v - 3.3v
 
+The argument of the constructor ADC specifies either a Pin by number, name of as
+Pin object, or a channel number in the range 0 - 3. If a pin is specified,
```

Comment:
I think it should also state you can pass in `ADC.CORE_TEMP` as a channel (channel 4).  That will configure the internal temperature sensor for reading on the ADC.

Reply:
Done.